pages/common/projucer.md

```diff
@@ -0,0 +1,28 @@
+# projucer
+
+> A project manager for JUCE framework applications.
+> More information: <https://juce.com/discover/stories/projucer-manual#10.4-command-line-tools>.
+
+- Display the help manual:
```

Comment:
Thanks for your contribution. Can you place the help command and its command description last as it is what we usually prefer?

Reply:
this should be better :)